articles/access_controls.md

```diff
@@ -1,62 +1,60 @@
-`Conditions for Use` determine how data can be used. `Conditions for Use` are determined based on nature of the data within a particular `file` or `table`. For example, a `file` or `table` may have `Conditions for Use` due to informed consent requirements, legal contracts, or other privacy requirements based on the source, collection method, and/or type of data in the file or table. It is important to note that `Conditions for Use` cannot be set for Synapse wiki; wiki are not designed to house data and therefore do not have `Conditions for Use` as a feature.
-
-**Users are responsible for determining the appropriate `Conditions for Use` for any data they upload into Synapse. **
+`Conditions for Use` indicate how content can be used. `Conditions for Use` are added to human data that require additional protection based on its type, source and/or collection method. For example, human 'omic' data may have `Conditions for Use` imposed by informed consent requirements, legal contracts, or other privacy requirements. It is also appropriate to add `Condition for Use` to data collected from "vulnerable" population or to content that could potentially harm individuals or groups if misused.
+**Controlled data may not be redistributed.** Controlled data can only be used by authorized investigators and are not transferable (unless explicitly specified otherwise). In other words, you cannot share Controlled data with collaborators unless they have been approved to use the data. Each Synapse users wishing to access Controlled Use data must individually agree to the `Conditions for Use` for that data.
+**Investigators are responsible for determining the appropriate `Conditions for Use` for any data they upload into Synapse.**
+It is important to note that `Conditions for Use` cannot be set for Synapse wiki or discussion forum. These are not designed to house data and therefore do not have local `Conditions for Use` as a feature. Synapse wikis and discussion forum inherit the condition for Use of their parent project.
 
 ### Data that does not require Conditions for Use
-Data in Synapse without `Conditions for Use` may be shared with all Synapse users. Typically these data are:  
-* Nonhuman data
-     * Human data that is:
-     * Publicly available elsewhere
-     * Anonymous data
-     * De-identified data with no known sharing or use restrictions
-     * Self-contributed data unambiguously consented for open data sharing
-_It is important to note that `Conditions for Use` cannot be set for Synapse `wiki`; `wiki` are not designed to house data and therefore do not have `Conditions for Use` as a feature._
+
+Open Data in Synapse are data without `Conditions for Use.` They can be used by any registered Synapse users authorized to view the project/folder/file/table (based on sharing setting) for research purpose. Typically Open Data are:
+* Data from model organism, species or strain
+* Non-biological data such as presentations, results or images
 
```

Comment:
What happened to 

+* Human data that are:
 +* Publicly available elsewhere
 +* Anonymous data
 +* De-identified non-sensitive data with no known sharing or use restrictions
 +* Self-contributed data unambiguously consented for open data sharing and use